profiles/kentik_snmp/cisco/cisco-firepower.yml

```diff
@@ -125,6 +125,20 @@ metrics:
         OID: 1.3.6.1.4.1.9.9.392.1.1.2.0
       - name: crasNumSetupFailInsufResources
         OID: 1.3.6.1.4.1.9.9.392.1.4.1.3.0
+        
+ # FTD failover status
+  - MIB: CISCO-UNIFIED-FIREWALL-MIB
+    symbols:
+      - name: cufwFOGroupIndex
+        OID: 1.3.6.1.4.1.9.9.491.1.4.2.1.1.1
+      - name: cufwFOGrpLastFailoverAt
+        OID: 1.3.6.1.4.1.9.9.491.1.4.2.1.1.2
+      - name: cufwFOGrpHAstate
+        OID: 1.3.6.1.4.1.9.9.491.1.4.2.1.1.3
+      - name: cufwFOGrpUpTime
+        OID: 1.3.6.1.4.1.9.9.491.1.4.2.1.1.4
+      - name: cufwFOGrpContextCount
+        OID: 1.3.6.1.4.1.9.9.491.1.4.2.1.1.5 
```

Comment:
This is a tabular OID, not a scalar; please adjust your PR to reflect the proper syntax referenced in the [template YAML file](https://github.com/kentik/snmp-profiles/blob/c19a259debbe8a5f314dc5470e5eda1fc540cd0b/profiles/kentik_snmp/_template.yml#L41-L79).

notes: ([reference docs](https://oid-rep.orange-labs.fr/get/1.3.6.1.4.1.9.9.491.1.4.2.1.1) since the Cisco link requires a login)
* `cufwFOGroupIndex` - index values are always collected on MIB tables and stored as `Index` on the metrics; this entry is not needed
* [cufwFOGrpLastFailoverAt](https://oid-rep.orange-labs.fr/get/1.3.6.1.4.1.9.9.491.1.4.2.1.1.2) - this is a string value, you'll need to use the `to_one` [conversion option](https://github.com/kentik/snmp-profiles/blob/c19a259debbe8a5f314dc5470e5eda1fc540cd0b/profiles/kentik_snmp/_template.yml#L98) here.
* [cufwFOGrpHAstate](https://oid-rep.orange-labs.fr/get/1.3.6.1.4.1.9.9.491.1.4.2.1.1.3) - this metric will need enumeration, it's listed as a `HardwareStatus` type, which is pulled from the `CISCO-FIREWALL-MIB`
* [cufwFOGrpUpTime](https://oid-rep.orange-labs.fr/get/1.3.6.1.4.1.9.9.491.1.4.2.1.1.4)
* [cufwFOGrpContextCount](https://oid-rep.orange-labs.fr/get/1.3.6.1.4.1.9.9.491.1.4.2.1.1.5)

Reply:
@thezackm , thanks for the inputs, made the suggested changes, please review.